examples/index.md

```diff
@@ -246,15 +219,7 @@ Content-Type: application/vnd.api+json
     {
       "status": "403",
       "source": { "pointer": "data/attributes/secret-powers" },
-      "detail": "Access to secret powers not authorized on Sundays."
-    },
-    {
-      "status": "404",
```

Comment:
I removed this example because it's an incorrect use of the `about` link, which is supposed to give a uri for this _particular occurence_ of the problem (so something like `http://example.com/errors/some-uuid`.) Moreover, I'm not sure whether 404 would've been the right response code (would depend, I guess, on the request).

I know you wanted a 404 example, so I'd be fine with adding something back in here, we just have to make sure it makes sense.